cypress/e2e/pages/agreeTerms.cy.ts

```diff
@@ -104,11 +104,10 @@ describe('Application', () => {
       applicationWithMainApplicant
     );
 
-    const errorStatus = 'Bad Request';
     const errorStatusCode = StatusCodes.BAD_REQUEST;
 
     //based on current API layer setup
-    const expectedErrorMessage = `Unable to update application: ${errorStatus} (${errorStatusCode})`;
+    const expectedErrorMessage = `Unable to update application (${errorStatusCode})`;
```

Comment:
Just from a FE perspective, I wonder if it would be worth having a generic message here i.e.

If you need help with this page please contact.  

Just thinking it very much like sorry - bad luck.